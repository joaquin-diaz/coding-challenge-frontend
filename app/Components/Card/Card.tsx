import styled from '@emotion/styled';

export default styled.div`
  background: white;
  box-shadow: 0 1px 6px 0 rgba(32, 33, 36, 0.28);
  box-sizing: border-box;
  border-radius: 5px;
`;
