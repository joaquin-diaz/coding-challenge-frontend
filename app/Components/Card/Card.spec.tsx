import * as React from 'react';

import Card from './Card';
import { mount } from 'enzyme';

describe('<Card />', () => {
  it('should render', () => {
    const wrapper = mount(<Card>Test</Card>);
    expect(wrapper).toMatchSnapshot();
  });
});
