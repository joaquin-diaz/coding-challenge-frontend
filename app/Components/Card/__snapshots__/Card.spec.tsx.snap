// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> should render 1`] = `
.emotion-0 {
  background: white;
  box-shadow: 0 1px 6px 0 rgba(32,33,36,0.28);
  box-sizing: border-box;
  border-radius: 5px;
}

<Styled(div)>
  <div
    className="emotion-0"
  >
    Test
  </div>
</Styled(div)>
`;
